ornl/summit/summit.config.yaml

```diff
@@ -16,7 +16,7 @@
 config:
   # This is the path to the root of the Spack install tree.
   # You can use $spack here to refer to the root of the spack instance.
-  install_tree: /gpfs/alpine/csc322/world-shared/spack-install/packages
+  install_tree: /ccs/home/xmeng/hpctoolkit-install/packages 
```

Comment:
I don't think that you want to put this in /ccs/home. This is an NFS file system. I think that you want the install directory to be in /gpfs. I think we have a project that you are part of where there is project space in /gpfs. Without being logged into summit, I am not sure where that is.

Reply:
@jmellorcrummey I think you are referring to  `/gpfs/alpine/csc322/world-shared`, which has the following permission setting:

`drwxrwsr-x  3 johnmc csc322   4096 Apr 26  2019 hpctoolkit`

I cannot create new directory there, so it looks like that I am not in the `csc322` user group.